### PR TITLE
PMM-7410 use grafana api for operations with folders

### PR DESCRIPTION
**Pull request by adivinho**
- [ ] https://github.com/Percona-Lab/pmm-submodules/pull/1882

Here are used two initial commits from 
https://github.com/percona/pmm-server/pull/248

The initial PR is broken after rebase

### CODE REVIEW

File: import-dashboards.py [hunk 1]
```diff
@@ -375,46 +375,46 @@ def add_folders(api_key):
...
+
+            compare_pattern = re.compile(r'^.*_Compare$')
+            compare_tags  = [s for s in item['tags'] if compare_pattern.match(s)]
+            ha_pattern = re.compile(r'^.*_HA$')
+            ha_tags  = [s for s in item['tags'] if ha_pattern.match(s)]
+
+            if len(compare_tags) > 0:
+                match_compare = re.match('(MySQL|PostgreSQL|MongoDB|OS)', compare_tags[0]);
+                if match_compare:
+                    print '     * Compare dashboard is detected for the service %s' % match_compare.group(0);
```
Comment (askomorokhov) [inline import-dashboards.py:403]:
```suggestion
                    print '     * Compare dashboard is detected for the service %s' % (match_compare.group(0),);
```

please use tuples for string formating

File: import-dashboards.py [hunk 2]
```diff
@@ -375,46 +375,46 @@ def add_folders(api_key):
...
+                    tag = match_compare.group(0);
+
+            if len(ha_tags) > 0:
+                match_ha = re.match('(MySQL|PostgreSQL|MongoDB)', ha_tags[0]);
+                if match_ha:
+                    print '     * HA dashboard is detected for the service %s' % match_ha.group(0);
```
Comment (askomorokhov) [inline import-dashboards.py:409]:
```suggestion
                    print '     * HA dashboard is detected for the service %s' % (match_ha.group(0),);
```

File: import-dashboards.py [hunk 3]
```diff
@@ -375,46 +375,46 @@ def add_folders(api_key):
...
+                    tag = match_ha.group(0);
+
+            r = requests.get('%s/api/dashboards/uid/%s' % (HOST,item['uid']), headers=grafana_headers(api_key))
+            dash_data = r.json()
+            dash_data['folderId'] = SET_OF_TAGS[tag]
+            dash_data['overwrite'] = True
+            r = requests.post('%s/api/dashboards/db' % (HOST), headers=grafana_headers(api_key), data=json.dumps(dash_data), verify=False)
```
Comment (askomorokhov) [inline import-dashboards.py:416]:
```suggestion
            r = requests.post('%s/api/dashboards/db' % (HOST,), headers=grafana_headers(api_key), data=json.dumps(dash_data), verify=False)
```

missed comman in tuple `(HOST,)`

**Review by askomorokhov — CHANGES_REQUESTED** [review]
Please fix minor issues.

**Review by askomorokhov — APPROVED** [review]
Cool!  😎 Looks good to me! 👍

**Review by BupycHuk — APPROVED** [review]

**Review by atymchuk — APPROVED** [review]

**Review by nikita-b — APPROVED** [review]

**Pull request merged**